### PR TITLE
Assignment11_Prem

### PR DESCRIPTION
**Pull request by Premlata1726**
image

### CODE REVIEW

File: src/prem/MaxMinNumber.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+			System.out.println(num1 + " is smaller between "+ num1 +" and " + num2);
+		}else {
+			System.out.println(num2 + " is bigger between "+ num1 +" and " + num2);
+		}
+	}
+	void max(int num1, int num2) {
+		if (num1> num2) {
+			System.out.println(num1 + " is bigger between "+ num1 +" and " + num2);
+		}else {
+			System.out.println(num2 + " is smaller between "+ num1 +" and " + num2);
```
Comment (AnvitP) [inline src/prem/MaxMinNumber.java:16]: else statement should be like System.out.println(num2 + " is bigger between "+ num1 +" and " + num2);

Comment (Premlata1726) [reply src/prem/MaxMinNumber.java:16]: @AnvitP Sorry but not getting your pint here, in above highlighted screenshot num2 is smaller than num1 but you had asked to mentioned below:
else statement should be like System.out.println(num2 + " is bigger between "+ num1 +" and " + num2);

Could you please elaborate it

Comment (AnvitP) [reply src/prem/MaxMinNumber.java:16]: change the print statement to System.out.println(num2 + " is bigger between "+ num1 +" and " + num2);

File: src/prem/MaxMinNumber.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package prem;
+
+public class MaxMinNumber {
+	
+	void min(int num1,int num2) {
+		if (num1<num2) {
+			System.out.println(num1 + " is smaller between "+ num1 +" and " + num2);
+		}else {
+			System.out.println(num2 + " is bigger between "+ num1 +" and " + num2);
```
Comment (AnvitP) [inline src/prem/MaxMinNumber.java:9]: change the else print statement to System.out.println(num1 + " is smaller between "+ num1 +" and " + num2);

**Review by Premlata1726 — COMMENTED** [review]
image

Corrected the else statement, please review.

**Pull request merged**